stacks/proxy.prx.org.yml

```diff
@@ -16,29 +16,51 @@ Parameters:
     Type: String
   EnvironmentTypeAbbreviation:
     Type: String
-  WildcardCertificateArn:
-    Type: String
 Mappings:
   EnvironmentTypeMap:
     Testing:
       DomainName: "proxy.test.prx.tech"
       HostedZoneName: "prx.tech."
+      ValidationDomain: "prx.tech"
       CorporateHost: "corporate.prx.tech"
       ExchangeHost: "exchange.test.prx.tech"
       PrxSessionKey: "_prx_session_test"
     Staging:
       DomainName: "proxy.staging.prx.tech"
       HostedZoneName: "prx.tech."
+      ValidationDomain: "prx.tech"
       CorporateHost: "corporate.prx.tech"
       ExchangeHost: "exchange.staging.prx.tech"
       PrxSessionKey: "_prx_session_staging"
     Production:
       DomainName: "proxy.prx.org"
       HostedZoneName: "prx.org."
+      ValidationDomain: "prx.org"
       CorporateHost: "corporate.prx.tech"
       ExchangeHost: "exchange.prx.org"
       PrxSessionKey: "_prx_session"
 Resources:
+  Certificate:
+    Type: "AWS::CertificateManager::Certificate"
+    Properties:
+      DomainName: !FindInMap [EnvironmentTypeMap, !Ref EnvironmentType, DomainName]
+      DomainValidationOptions:
+        - ValidationDomain: !FindInMap [EnvironmentTypeMap, !Ref EnvironmentType, ValidationDomain]
+          DomainName: !FindInMap [EnvironmentTypeMap, !Ref EnvironmentType, DomainName]
+      SubjectAlternativeNames:
+        - "prx.org"
+        - "www.prx.org"
+      Tags:
+        - Key: Project
+          Value: Corporate
```

Comment:
`proxy.prx.org`

Reply:
should i change the tag on the lambda function as well then?

Reply:
Yes please